worlds_nif/nif-haiti.py

```diff
@@ -0,0 +1,111 @@
+from random import randint
+
+# FORMATTING
+#############
+
+
+def remove_symbols(dirty_nif):  # type: (str) -> str
+    """
+    Removes spaces, dots, dashes, and other symbols from a NIF.
+
+    Args:
+        dirty_nif (str): A string containing a NIF with possible non-numeric characters.
+
+    Returns:
+        str: A cleaned NIF containing only numeric digits.
+
+    Example:
+        remove_symbols("123-456-7890") -> "1234567890"
+    """
+    return "".join(filter(str.isdigit, dirty_nif))
+
+
+def format_nif(nif):  # type: (str) -> str
+    """
+    Formats a Haitian NIF for display with visual aid symbols.
+
+    This function formats a numbers-only NIF string by adding dashes
+    for better readability, commonly used in Haitian documents.
+
+    Args:
+        nif (str): A numbers-only NIF string of exactly 10 digits.
+
+    Returns:
+        str: A formatted NIF string with dashes, or None if the input is invalid.
+
+    Example:
+        format_nif("1234567890") -> "123-456-7890"
+    """
+    # Clean the NIF first, in case it contains symbols
+    nif = remove_symbols(nif)
+
+    # Ensure the NIF is valid for formatting
+    if len(nif) != 10:
+        return None
+
+    # Add dashes after the third and sixth digits
+    return "{}-{}-{}".format(nif[:3], nif[3:6], nif[6:])
+
+
+# OPERATIONS
+#############
+
+
+def validate(nif):  # type: (str) -> bool
```

Comment:
Aqui, para seguir o padrão dos nomes de métodos, o nome deve ser is_valid. Ficaria assim:

```suggestion
def is_valid(nif):  # type: (str) -> bool
```